app/scripts/lib/user-agent.js

```diff
@@ -67,6 +67,20 @@ define(function (require, exports, module) {
       },
 
       /**
+       * Check if the current browser has web channel support.
+       *
+       * @returns {Boolean}
+       * @private
+       */
+      _hasWebChannelSupport () {
```

Comment:
The typical convention in JavaScript is that methods prefixed with an underscore like this signify a private method. Since it used outside of this module, it makes sense to not have the underscore prefix.